### ISSUE TITLE
Connect TurtleBot network to the internet

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#14]
## User story
1. As a SD
2. I want to connect the TurtleBot network to the internet
3. So that I can download additional ROS packages on the TurtleBot and Google stuff on my notebook.

## Acceptance criteria
* Connection is established between TurtleBot and internet

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Sebastian2023** [comment]
Solved using OpenWRT. The two TurtleBots connected to the TurtleBotStation.

![Sick_AMOS_network](https://user-images.githubusercontent.com/92729241/168662599-6279ba04-8de3-46ec-9360-946d0814296c.jpg)

https://openwrt.org/
